integtest/src/main/scala/com/pingcap/spark/IssueTestCase.scala

```diff
@@ -39,17 +39,64 @@ class IssueTestCase(prop: Properties) extends TestCase(prop) {
     refresh()
     result |= execBothAndJudge("select * from t")
     result = !result
-    logger.warn(s"\n*************** Index Tests result: $result\n\n\n")
+    logger.warn(s"\n*************** Issue Client#0198 Tests result: $result\n\n\n")
+  }
+
+  private def TestSpark0160(dbName: String): Unit = {
+    var result = false
+    jdbc.execTiDB(s"use $dbName")
+    jdbc.execTiDB("drop table if exists t")
+    jdbc.execTiDB("create table t(c1 int not null, c2 int not null)")
+    jdbc.execTiDB("insert into t values(4, 2)")
+    jdbc.execTiDB("insert into t values(1, 4)")
+    jdbc.execTiDB("insert into t values(3, 3)")
+    jdbc.execTiDB("insert into t values(3, 4)")
+    refresh() // refresh since we need to load data again
+    result |= execAllAndJudge("select * from t order by c1, c2")
+    result |= execAllAndJudge("select * from t order by c1 asc, c2 asc")
+    result |= execAllAndJudge("select * from t order by c1 asc, c2 desc")
+    result |= execAllAndJudge("select * from t order by c1 desc, c2 asc")
+    result |= execAllAndJudge("select * from t order by c1 desc, c2 desc")
+    result |= execAllAndJudge("select * from t order by (c1, c2)")
+    result |= execAllAndJudge("select * from t order by (c1, c2) asc")
+    result |= execAllAndJudge("select * from t order by (c1, c2) desc")
+    result = !result
+    logger.warn(s"\n*************** Issue Spark#0162 Tests result: $result\n\n\n")
```

Comment:
162->160

Reply:
thx for pointing oit

Reply:
BTW, coding on the subway might not be a good habit. LOL :laughing: